Blankmon/src/Enums/Abilities.java

```diff
@@ -0,0 +1,5 @@
+package Enums;
+
+public enum Ability {
```

Comment:
Can you change Ability to Abilities?

I have a HashMap I am working on and need the change for naming reasons.

I need to do HashMap<Abilities, Ability>.